task/upload.go

```diff
@@ -528,23 +522,6 @@ func isHLSFile(fname string) bool {
 	return fname[ext:] == ".m3u8"
 }
 
-func removeCredentials(metadata *clients.CatalystCallback) *clients.CatalystCallback {
```

Comment:
Was this only used when saving that `video.json` file?

Reply:
Yep 👍 no more usages of it now